test/e2e/artifact_test.go

```diff
@@ -115,9 +115,10 @@ var _ = Describe("Podman artifact", func() {
 		Expect(a.Manifest.Layers[0].Annotations["flavor"]).To(Equal("lemon"))
 		Expect(a.Manifest.Layers[0].MediaType).To(Equal(yamlType))
 
-		failSession := podmanTest.Podman([]string{"artifact", "add", "--annotation", "org.opencontainers.image.title=foobar", "foobar", artifact1File})
-		failSession.WaitWithDefaultTimeout()
-		Expect(failSession).Should(ExitWithError(125, "Error: cannot override filename with org.opencontainers.image.title annotation"))
+		title := RandomString(12)
+		annotation3 := fmt.Sprintf("org.opencontainers.image.title=%s", title)
+		podmanTest.PodmanExitCleanly("artifact", "add", "--annotation", annotation3, "foobar", artifact1File)
+		Expect(a.Manifest.Layers[0].Annotations["org.opencontainers.image.title"]).To(Equal(title))
```

Comment:
that is not checking the right thing, you need to inspect the artifact you just created with podmanTest.InspectArtifact("foobar") and use that return value to check the annotation